src/js/Controllers/UIController.js

```diff
@@ -21,6 +21,20 @@ import store from '../store'
 import sdlController from './SDLController'
 import SubmenuDeepFind from '../Utils/SubMenuDeepFind'
 
+const getNextSystemContext = () => {
+    const state = store.getState();
+    const activeApp = state.activeApp;
+    const pathName = window.location.pathname;
```

Comment:
```suggestion
    const pathName = window.location.hash;
```

you could potentially use === with the hash property